### PR TITLE
updates based on feedback

### PR DESCRIPTION
**Pull request by abgeorge7**

### CODE REVIEW

File: src/components/UserInformation.js [hunk 1]
```diff
@@ -94,6 +110,16 @@ class UserInformation extends React.Component {
     }
   }
 
+  validate = () => {
+    if (this.state.usernameOption) {
```
Comment (paulineribeyre) [inline src/components/UserInformation.js]: if i click the "add user" button without selecting an option in the dropdown, the form is submitted without validation

File: src/components/FormInformation.css [hunk 2]
```diff
@@ -31,6 +31,31 @@
...
+  padding: 0;
+  width: 80%;
+  margin-left: 14px;
+  font-size: 14px;
+  outline: none;
+}
+
+.form-info__detail-select__control {
+  border-radius: 4px;
+  border: 1px solid var(--g3-color__gray) !important;
```
Comment (abgeorge7) [inline src/components/FormInformation.css]: why !important? I try to avoid if possible :P

File: src/components/Homepage.js [hunk 3]
```diff
@@ -1,13 +1,27 @@
...
+    };
+  }
+
   componentDidMount() {
     userIsLoggedIn(this.props.token).then((loggedIn) => {
       if (loggedIn) {
         this.props.history.push('/manage');
+      } else {
+        fetchLoginOptions().then((options) => {
+          console.log(options)
```
Comment (abgeorge7) [inline src/components/Homepage.js]: remove console.log

File: src/components/UserInformation.js [hunk 4]
```diff
@@ -113,19 +133,28 @@ class UserInformation extends React.Component {
   }
 
   render() {
-    console.log('this.props', this.props)
+    console.log('state', this.state);
```
Comment (abgeorge7) [inline src/components/UserInformation.js]: remove console

**Pull request merged**